File: Weight Completion Service/main/common/logger.py
from datetime import datetime


def log(message):
    _log("Info", message)


def warning(message):
    _log("WARNING", message)


def _log(error_type, message):
    t_string = datetime.now().strftime("%d/%m/%Y %H:%M:%S")
    print(f"{t_string}    {error_type}    {message}")

File: Map Matching Service/src/main/java/com/mapmatching/common/Logger.java
package com.mapmatching.common;

import java.text.SimpleDateFormat;
import java.util.Date;

import com.fasterxml.jackson.databind.deser.ValueInstantiator.Gettable;

// Provies methods to perform simple logging.
public class Logger {

  public static void Info(String message) {
    Log("[INFO]", message);
  }

  public static void Warning(String message) {
    Log("[WARNING]", message);
  }

  public static void Error(String message) {
    Log("[ERROR]", message);
  }

  public static void Log(String errorType, String message) {
    System.out.println(errorType + "      " + GetDateTimeString(new Date()) + "     " + message);
  }

  // Returns a date in a beautiful format.
  public static String GetDateTimeString(Date date) {
    SimpleDateFormat formatter = new SimpleDateFormat("dd/MM/yyyy HH:mm:ss");
    return formatter.format(date);
  }
}
